2024-01-15 08:21:06,437 INFO    MainThread:460 [wandb_setup.py:_flush():76] Configure stats pid to 460
2024-01-15 08:21:06,438 INFO    MainThread:460 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hugoa\.config\wandb\settings
2024-01-15 08:21:06,438 INFO    MainThread:460 [wandb_setup.py:_flush():76] Loading settings from c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\settings
2024-01-15 08:21:06,438 INFO    MainThread:460 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-15 08:21:06,438 INFO    MainThread:460 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-15 08:21:06,438 INFO    MainThread:460 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-15 08:21:06,438 INFO    MainThread:460 [wandb_init.py:_log_setup():506] Logging user logs to c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\run-20240115_082106-mxbttkxl\logs\debug.log
2024-01-15 08:21:06,439 INFO    MainThread:460 [wandb_init.py:_log_setup():507] Logging internal logs to c:\Users\hugoa\H9\RL\GitHub Vrain\rl_levitation_control\rl_levitation_control\wandb\run-20240115_082106-mxbttkxl\logs\debug-internal.log
2024-01-15 08:21:06,439 INFO    MainThread:460 [wandb_init.py:init():546] calling init triggers
2024-01-15 08:21:06,439 INFO    MainThread:460 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2024-01-15 08:21:06,439 INFO    MainThread:460 [wandb_init.py:init():568] re-initializing run, found existing run on stack: 0wxu2w01
2024-01-15 08:21:06,440 INFO    MainThread:460 [wandb_run.py:_finish():1854] finishing run hyperloopupv/2gdl/0wxu2w01
2024-01-15 08:21:06,446 ERROR   MainThread:460 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-01-15 08:21:06,446 INFO    MainThread:460 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-15 08:21:06,446 INFO    MainThread:460 [wandb_init.py:_jupyter_teardown():434] cleaning up jupyter logic
2024-01-15 08:21:06,446 INFO    MainThread:460 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2024-01-15 08:21:06,446 INFO    MainThread:460 [wandb_run.py:_restore():2066] restore
2024-01-15 08:21:06,446 INFO    MainThread:460 [wandb_run.py:_restore():2072] restore done
